DFC.App.DiscoverSkillsCareers.Services/SessionIdToCodeConverters/SessionIdToCodeConverter.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DFC.App.DiscoverSkillsCareers.Services.Contracts;
+using System.Text;
+
+namespace DFC.App.DiscoverSkillsCareers.Services.SessionIdToCodeConverters
+{
+    public class SessionIdToCodeConverter : ISessionIdToCodeConverter
+    {
+        public string GetCode(string value)
+        {
+            var result = new StringBuilder();
+            if (!string.IsNullOrWhiteSpace(value))
+            {
+                value = value.Trim().ToUpper();
+                int i = 0;
+                foreach (var c in value)
```

Comment:
In this case a standard for loop would of been better.

Reply:
Modified to use for loop